Frameworks/UIKit/UITouch.mm

```diff
@@ -225,7 +226,14 @@ - (CGPoint)locationInView:(UIView*)viewAddr {
     if (viewAddr != 0) {
         UIWindow* window = [viewAddr window];
         if (window != nil) {
-            ret = [window _convertPoint:location fromView:window toView:viewAddr];
+            UIWindow* fromWindow = window;
+
+            if (GetCACompositor()->IsRunningAsFramework()) {
+                // In framework mode the parent UIWindow is not visible
+                fromWindow = nil;
+            }
+
+            ret = [window _convertPoint:location fromView:fromWindow toView:viewAddr];
```

Comment:
yeah, those convert methods on UIWindow don't even belong there - tracked by [https://github.com/Microsoft/WinObjC/issues/2440](https://github.com/Microsoft/WinObjC/issues/2440).  maybe we should just fix instance this to call UIView's point conversion method instead?

---
In reply to: [113979397](https://github.com/Microsoft/WinObjC/pull/2650#discussion_r113979397) [](ancestors = 113979397)